.github/workflows/move_to_in_review.yml

```diff
@@ -0,0 +1,39 @@
+name: Move PR to In Review
+on:
+  pull_request_target:
+    types:
+      - review_requested
+
+env:
+  PROJECT_ID: "PVT_kwDOAKRRkc4AfZil"
+  STATUS_FIELD_ID: "PVTSSF_lADOAKRRkc4AfZilzgUwVMs"
+  IN_REVIEW_ID: "618f8b62"
+  PR_ID: ${{ github.event.pull_request.number }}
+
+jobs:
+  move-to-in-review:
+    name: Move to In Review
+    runs-on: ubuntu-latest
+    steps:
+      - name: Generate token
+        id: generate-token
+        uses: actions/create-github-app-token@v1
+        with:
+          app-id: ${{ vars.APP_ID }}
+          private-key: ${{ secrets.APP_PEM }}
+
+      - name: Get PR project ID
+        id: get-pr-id
+        env:
+          GH_TOKEN: ${{ steps.generate-token.outputs.token }}
+        run: |
+          PR_PROJECT_ID="$(gh project item-list -L 9999 --owner Submitty 1 --format json --jq "[.items[] | {id, labels, status, title: .content.title, repo: .content.repository, number: .content.number}] | .[] | select(.number == "$PR_ID").id")"
+          echo "pr-project-id="${PR_PROJECT_ID:?}"" >> "$GITHUB_OUTPUT"
```

Comment:
This is messy, and bugs could easily be added in the future.  Wouldn't it be better to use the Python scripts you created for this?

Reply:
The Python script I made a while ago ended up being overengineered for a task like this, however it would be easy to put this in a shell script instead.